src/api/java/CMakeLists.txt

```diff
@@ -14,15 +14,15 @@
 ##
 
 # create directories
-file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/src/api/java/cvc5")
+file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/src/api/java")
 set(JNI_DIR "${CMAKE_CURRENT_BINARY_DIR}/src/api/java/jni")
 file(MAKE_DIRECTORY ${JNI_DIR})
 
 # Generate cvc5/Kind.java
 configure_file(genkinds.py.in genkinds.py)
 
 set(JAVA_KIND_FILE
-  "${CMAKE_CURRENT_BINARY_DIR}/src/api/java/cvc5/Kind.java"
+  "${CMAKE_CURRENT_BINARY_DIR}/src/api/java/Kind.java"
```

Comment:
Shouldn't this also go to `src/api/java/io/github/cvc5/api/`?

Reply:
Yes. Done